dashboard/test/integration/db_query_test.rb

```diff
@@ -152,7 +152,7 @@ def setup
     student.assign_script(unit)
     sign_in student
 
-    assert_cached_queries(16) do
+    assert_cached_queries(17) do
```

Comment:
@kobryan0619 this looks like what we were seeing

@etaderhold could you explain why we need this?

Reply:
Changing the `User.teachers` relationship to go through the `Section.instructors` relation causes one more SQL query to be generated than before. I don't know _why_ we have a set of tests that counts SQL queries and fails when that number changes, but we do. This increment is an expected result of this code change.